src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLogger.cs

```diff
@@ -29,12 +29,9 @@ internal static string GetKeyWithoutFirstSymbol(ConcurrentDictionary<string, str
     readonly SerilogLoggerProvider _provider;
     readonly ILogger _logger;
 
-    static readonly CachingMessageTemplateParser MessageTemplateParser = new();
+    readonly EventIdPropertyCache _eventPropertyCache = new (48);
```

Comment:
Since the cached properties are now keyed by both id and name, the maximum size of the cache might need to be increased, here.